seed/migrations/0119_column_recognize_empty.py

```diff
@@ -12,9 +12,9 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AddField(
-            model_name='column',
-            name='recognize_empty',
-            field=models.BooleanField(default=False),
-        ),
+        migrations.RunSQL(
+            'ALTER TABLE seed_column'
+            '    ADD COLUMN IF NOT EXISTS recognize_empty BOOLEAN NOT NULL DEFAULT FALSE;',
```

Comment:
this is fantastic. thanks for adding this! It will make migration much easier.

Reply:
All props to @axelstudios - he thought to use that query instead of what I had 😄